uprotocol/uattributes.proto

```diff
@@ -60,19 +60,20 @@ message UAttributes {
     // https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/basics/qos.adoc[uProtocol Prioritization]
     UPriority priority = 5;
 
-    // TTL is How long this message should live for after it was generated (in milliseconds).
+    // TTL is How long this message should live for after it was generated (in milliseconds). If not present,
+    // the message is assumed to not timeout (should live forever)
     // Event expires when:
     // \$t_current > t_{id} + ttl\$
-    optional int32 ttl = 6;
+    optional uint32 ttl = 6;
     
     // Permission level per https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/basics/permissions.adoc[Permissions]
-    optional int32 permission_level = 7;
+    optional uint32 permission_level = 7;
```

Comment:
Reading [thru the spec](https://github.com/eclipse-uprotocol/up-spec/blob/main/basics/permissions.adoc), I see that the default when missing is set to 5 if an Android app and if not, it appears that's zero:
> For Android we will use the AOSP-standard manifest extension mechanism of <meta-data> to describe the permission level. We’ll use the name of uprotocol.permissionLevel and the value of the actual level, 0 to 5. The default value of 5 will be assigned implicitly when uprotocol.entity.permissionLevel` is not declared. 

> uPL value MUST be an integer, 0 to N, where N is the most permissive level. The default value is 0.

Is that why a default is not outlined here if not present? It's kind of implementation / host device dependent?

Reply:
investigating now... Great catch thank you @PLeVasseur !

Reply:
Was this resolved? I think @stevenhartley was looking into it.